src/opcode.rs

```diff
@@ -5,10 +5,16 @@
 use std::mem;
 use std::os::unix::io::RawFd;
 
-use crate::squeue::Entry;
+use crate::squeue::{Entry, OptionValues};
 use crate::sys;
 use crate::types::{self, sealed};
 
+/// Trait to prepare an SQE from an opcode object.
+pub trait PrepareSQE {
+    /// Prepare an SQE from an opcode object.
+    fn prepare(&self, sqe: &mut sys::io_uring_sqe, options: Option<&OptionValues>);
```

Comment:
It should not expose internal types, maybe this trait can not be exposed。

Reply:
Fixed